scripts/microbe_stage/organelle.as

```diff
@@ -722,6 +722,13 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
                             Ogre::Vector3(0, 0, 1)));
         // Add hex collision shapes
         auto hexes = organelle.getHexes();
+        
+        //! Offset depending if species is bacteria or not
+        float hexOffset = 0;   
```

Comment:
This isn't applied to the position of the hex. There should be a separate variable for the hex size and offset amount.